src/utils/url.ts

```diff
@@ -15,8 +15,7 @@ export function isEmail(email: string): boolean {
 }
 
 export function getRegistryURL(): string {
-  // Don't add slash if it's not a sub directory
-  return `${location.origin}${location.pathname === '/' ? '' : location.pathname}`;
+  return (window as any).__VERDACCIO_BASENAME_UI_OPTIONS.base;
```

Comment:
```suggestion
  return window.__VERDACCIO_BASENAME_UI_OPTIONS.base;
```

you don't need to cast window as any here, since there is already a type __VERDACCIO_BASENAME_UI_OPTIONS for window in the file types/index.ts. Please see: 
![image](https://user-images.githubusercontent.com/29228205/69481050-f1a97600-0e0d-11ea-9a45-18cba7118598.png)

Reply:
Okies! I will update that.